.prettierignore

```diff
@@ -0,0 +1,2 @@
+bazelrc
+**/*.md
```

Comment:
Let's add the pnpm lockefile to this and revert there change to them

Reply:
Got it, done.

Reply:
Should we only ignore the lockfiles and not all yaml though?

Reply:
Sure, updated.